pkg/reconciler/instances/istio/istio.go

```diff
@@ -38,7 +38,6 @@ func init() {
 		WithPreReconcileAction(NewStatusPreAction(istioPerformerCreatorFn)).
 		WithReconcileAction(NewIstioMainReconcileAction(istioPerformerCreatorFn)).
 		WithPostReconcileAction(actions.NewActionAggregate(
-			NewMutatingWebhookPostAction(istioPerformerCreatorFn),
```

Comment:
Did you remove this action and its tests?

Reply:
yes, tests are adjusted